tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -283,8 +283,16 @@ function kbs_k8s_deploy() {
 	echo "somesecret" > overlays/$(uname -m)/key.bin
 
 	# For qemu-se runtime, prepare the necessary resources
-	if [ "${KATA_HYPERVISOR}" == "qemu-se" ]; then
-		prepare_credentials_for_qemu_se
+	if [ "$(uname -m)" == "s390x" ]; then
+		if [ "${KATA_HYPERVISOR}" == "qemu-se" ]; then
+			prepare_credentials_for_qemu_se
+		elif [ "${KATA_HYPERVISOR}" == "qemu-coco-dev" ]; then
+			# Create an empty directory just for deployment
+			export IBM_SE_CREDS_DIR="$(mktemp -d -t ibmse.creds.XXXXXXXXXX)"
```

Comment:
Out of interest, for non-SE s390x, what's the reason for `IBM_SE_CREDS_DIR` being needed? Is this something that we could improve in Trustee at some point so non-TEE s390x is tolerated better?

Reply:
Yes, you are right. We could improve this in Trustee (https://github.com/confidential-containers/trustee/blob/main/kbs/config/kubernetes/overlays/s390x/) by differentiating overlays. I think we could proceed this PR apart from that because it is all about tests (`qemu-coco-dev` is also designed only for tests).

Reply:
Agreed, which is why I approved, I was just interesting in the reasoning. Thanks